mlflow/cli.py

```diff
@@ -124,12 +124,14 @@ def run(uri, entry_point, version, param_list, experiment_id, mode, cluster_spec
 @click.option("--file-store-path", default=None,
               help="The root of the backing file store for experiment and run data. Defaults to %s."
                    % file_store._default_root_dir())
-def ui(file_store_path):
+@click.option("--host", default="127.0.0.1",
+              help="The networking interface on which the UI server listens. Defaults to 127.0.0.1.  Use 0.0.0.0 for docker.")
```

Comment:
Looks like this line (and the associated one) has a linter error due to line length. In general you can run `bash lint.sh` locally to run the linter.

Maybe we could change the help a bit while we're at it (mostly to clarify the implication of the default and of setting 0.0.0.0):

```python
@click.option("--host", default="127.0.0.1",
              help="The IP address which the UI server listens."
                   " Defaults to 127.0.0.1, allowing only local connections."
                   " Use 0.0.0.0 to bind to all addresses.")
```